scripts/ci/xcc-builds/build-osxcross-clang.sh

```diff
@@ -32,7 +32,7 @@ docker run --rm -i -v ${OUT_DIR}:/out -w /tmp ${BUILD_IMAGE} bash <<EOF
   git clone https://github.com/tpoechtrager/osxcross.git /usr/local/osxcross && \
     cd /usr/local/osxcross && \
     git checkout ${OSXCROSS_VERSION} && \
-    curl -L -o ./tarballs/MacOSX10.11.sdk.tar.xz https://macos-sdks.s3.amazonaws.com/MacOSX10.11.sdk.tar.xz && \
+    curl -L -o ./tarballs/MacOSX10.12.sdk.tar.xz https://storage.googleapis.com/influxdata-team-flux/macos-sdks/MacOSX10.12.sdk.tar.xz && \
```

Comment:
Does this mean we support fewer Mac versions now? That should be in the release notes

Reply:
Will add a note, good catch

Reply:
Done